elements.html

```diff
@@ -118,36 +125,18 @@ <h2 class="s-h2">Paragraph Text</h2>
     {% assign dark_texts = texts | where: "light_text", nil %}
 
     {% for text in dark_texts %}
-    <tr class="token-table-row">
-      <td class="element-code-cell">
-        <code>{{ text.value }}</code>
-      </td>
-      <td>
-        <code class="language-html" data-lang="html">
-          {{text | remove: '<p>' | remove: '</p>' | escape}}
-        </code>
-      </td>
-      </td>
-      <td class="element-sample-cell">
-        {{ text }}
-      </td>
+    <tr class="table-row">
```

Comment:
All these repeated tables are making me wonder if it's time to start abstracting them out into [Jekyll includes](https://jekyllrb.com/docs/includes/)

Reply:
Yes, me too. At least for the ones that are the same on each page, like all the tokens, etc.